### PR TITLE
Milestone #2 : Create calculator UI

### PR DESCRIPTION
**Pull request by ahmed025566**
In this milestone you could see that I have made the following: 

- I have created a component called Input which will display the result.
- I have created a button component then I have used it in the calculator component with changing some props like the inner html and the color.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,25 +1,11 @@
...
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <Calculator />
```
Comment (rustam-tolipov) [inline src/App.js:7]: **You have done a great job!**

However, in your app, we have some places to improve and fix. For starting, all the boilerplate that got created by the CRA (texts, images, and styles) should be deleted. Kindly delete them :)

Screenshot 2023-05-31 at 03 55 42

File: src/App.css [hunk 2]
```diff
@@ -1,39 +1,12 @@
...
 
-@keyframes App-logo-spin {
-  from {
-    transform: rotate(0deg);
-  }
-
-  to {
-    transform: rotate(360deg);
-  }
+.App {
```
Comment (rustam-tolipov) [inline src/App.css:7]: - [ ] Somehow, your app does not match the given wireframe. Please fix it as well :)


Your image            |  Original image
:-------------------------:|:-------------------------:
Screenshot 2023-01-03 at 18 29 21 |

File: src/components/calculator.js [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        >
+        {symbol}
+      </div>
+);
+
+const Calculator = () => (
+    <div className='calc-wrapper'>
+        <Input></Input>
+        <div className='raw'>
+        <Button symbol="A/c"/>
```
Comment (rustam-tolipov) [inline src/components/calculator.js:18]: - [ ] `Calculator.js` should have at least 1 child component. I suggest you make the `Button` component external and import it into `Calculator.js`

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hello @ahmed025566,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_
_No Linters errors_
_Great readme_
_Good design_

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Check the comments under the review._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by anagudelogu — APPROVED** [review]
# STATUS: APPROVED! 🍾 🎆 🥇 

Hi @ahmed025566, 
I'm Andres.

You did a great job working on this milestone, your project is complete! 🚀 
There is nothing else to say other than... it's time to merge it 🤖  
Congratulations! 🎉 



## Highlights
✅ Great use of Git flow. 💯 
✅ Your project is documented professionally. 📜 
✅ Great UI design. 💻 
✅ Descriptive PR title, summary and commit messages. 💬 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**